src/main/java/frc/robot/Robot.java

```diff
@@ -29,6 +36,27 @@ public void robotInit() {
         // autonomous chooser on the dashboard.
         robotContainer = new RobotContainer();
 
+        Logger.recordMetadata("ProjectName", "MyProject"); // Set a metadata value
+
+        if (isReal()) {
+            Logger.addDataReceiver(new WPILOGWriter()); // Log to a USB stick ("/U/logs")
+            Logger.addDataReceiver(new NT4Publisher()); // Publish data to NetworkTables
+            new PowerDistribution(1, ModuleType.kRev); // Enables power distribution logging
```

Comment:
This is only valid for the Rev PDH, i'd take it out of the template. Or comment it out